backend/app/services/llm/jobs.py

```
@@ -134,14 +135,48 @@ def execute_job(
-    )
+        f"[execute_job] Starting LLM job execution | job_id={job_id}, task_id={task_id}, "    )
 
     try:
+        if input_guardrails:
+            safe_input = call_guardrails(input_query, input_guardrails, job_id)
+
+            logger.info(
+                f"[execute_job] Input guardrail validation | success={safe_input['success']}."
+            )
+
+            if safe_input["bypassed"] == True:
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `safe_input["bypassed"]:` for truth checks

Replace with `safe_input["bypassed"]`

(E712)